### ISSUE TITLE
v4 colortbl now throws on columncolor in table arguments

### ISSUE THREAD

**Issue by pkra** [mathjax/MathJax#2955]
I would like to check if the following is an expected change in v4.

Take the following example.

```
\begin{array}{>{\columncolor[RGB]{216, 216, 255}} c >{\columncolor[RGB]{216, 216, 255}} c >{\columncolor[RGB]{216, 216, 255}} c} 0 & 1 & 0 \\0 & 0 & 1 \\0 & 0 & 0 \end{array} 
```

With v3, (e.g., mathjax.org#demo), the column color is ignored as documented in https://docs.mathjax.org/en/latest/input/tex/extensions/colortbl.html.

With v4, I get an error `\columncolor must be in the top row`.

**Comment by dpvc** [comment]
Of course, in v3 the `>{...}` declaration was ignored, so that is the reason for the documentation you site.  This will have to be changed in v4, and the message should be removed.
